Decred Wallet/view_controller/ReceiveViewController.swift

```diff
@@ -15,17 +15,29 @@ class ReceiveViewController: UIViewController,UIDocumentInteractionControllerDel
 
     @IBOutlet weak var subheader: UILabel!
     @IBOutlet weak var lblWalletAddress: UILabel!
+    @IBOutlet var contentStackView: UIStackView!
     
     private var barButton: UIBarButtonItem?
-    
+    private lazy var syncInProgressLabel: UILabel = {
+        let label = UILabel()
+        label.translatesAutoresizingMaskIntoConstraints = false
+        label.textColor = .darkGray
+        label.numberOfLines = 0
+        label.textAlignment = .center
+        label.text = "Please wait for your wallet to finish synchronizing."
+        return label
+    }()
+
     var firstTrial = true
     var starttime: Int64 = 0
     var myacc: WalletAccount!
     var account: WalletAccounts?
     var tapGesture = UITapGestureRecognizer()
     var oldAddress = ""
     var wallet = AppDelegate.walletLoader.wallet
-    
+    let isSynced = AppDelegate.walletLoader.isSynced
```

Comment:
We're not really particularly interested in the status of the current sync operation. We're more interested in knowing if this wallet has been successfully/completely synced at least once, possibly in a previous run of the app.

We should only show a "wait for sync" label, if the wallet was restored, and it has **NEVER** been synced successfully/completely.